packages/exhale/build.sh

```diff
@@ -0,0 +1,9 @@
+TERMUX_PKG_HOMEPAGE=https://gitlab.com/ecodis/exhale
+TERMUX_PKG_DESCRIPTION="Open source xHE-AAC encoder"
+TERMUX_PKG_LICENSE="custom"
+TERMUX_PKG_LICENSE_FILE="include/License.htm"
+TERMUX_PKG_MAINTAINER="@DLC01"
+TERMUX_PKG_VERSION=1.1.6
+TERMUX_PKG_SRCURL=https://gitlab.com/ecodis/exhale/-/archive/v${TERMUX_PKG_VERSION}/exhale-v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=bf817c42866294ce1c3187d5b09d9e48e46d36ef2ba8ff3b7630e87c5113ea9d
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="-DBUILD_SHARED_LIBS=ON, -DCMAKE_BUILD_TYPE=Release"
```

Comment:
The comma doesn't cause problems?

Reply:
i don't think so it didn't cause me any problems

Reply:
This is passed to [a command-line CMake invocation](https://github.com/termux/termux-packages/blob/679c8645ed1c60629661ba3ed14a5bc87cf3c5ed/scripts/build/configure/termux_step_configure_cmake.sh#L54), where commas can cause problems, so remove it just in case.

Reply:
My guess is that CMake accepts `ON,` as equivalent to `ON` but other build configuration systems won't, and CMake might stop one day too.